pkg/interface/src/views/apps/launch/components/tiles/weather.js

```diff
@@ -248,12 +230,11 @@ export default class WeatherTile extends React.Component {
           display="flex"
           flexDirection="column"
         >
-          <Text color={weatherStyle.text}>{c.summary}</Text>
-          <Text color={weatherStyle.text}>{Math.round(c.temperature)}°</Text>
-          <Text color={weatherStyle.text}>{nextSolarEvent}</Text>
+          <Text>{c.weatherDesc[0].value.replace(/([a-z])([A-Z])/g, '$1 $2')}</Text>
+          <Text>{Math.round(c.temp_F)}℉</Text>
```

Comment:
We were doing `?=auto` to get Celsius in ~most of the world~ countries that use it. Any way to maintain that here?

Reply:
Well both units are in there. We could just check if the user ~~is free~~ location is in the United States and switch on that

Reply:
There's a few others but seems like an ok heuristic for now.